### ISSUE TITLE
Cover mode appears disabled after launching any activity

### ISSUE THREAD

**Issue by izzel** [otakuhqz/Omega#90]
## Description

Cover mode appears disabled after launching an activity, but shows that it is enabled in the group settings. I think this was working in 0.7.1.

## Expected Behaviour

After coming back from an activity, group should remember it's in cover mode.

## Actual Behaviour
Cover mode appears as if it is disabled and Omega will crash if long pressed.

## Steps to Reproduce

1. Create cover mode group
2. Launch any activity and come back to home
3. App group appears as if cover mode is not enabled
4. Long pressing group will crash app

## Device and OS

* Device: OnePlus 5t
* Android version: 9
* Launcher version: 0.8.0
* Rom: Stock

## Logcat

```
Build version: 0.8.0 
Build date: 1981-01-01 01:01:02 
Current date: 2021-06-12 21:28:07 
Device: OnePlus ONEPLUS A5010 
OS version: Android 9 (SDK 28) 
 
Stack trace:  
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.graphics.drawable.Drawable.copyBounds(android.graphics.Rect)' on a null object reference
	at com.android.launcher3.graphics.DragPreviewProvider.getDrawableBounds(DragPreviewProvider.java:152)
	at com.android.launcher3.graphics.DragPreviewProvider.createDragBitmap(DragPreviewProvider.java:133)
	at com.android.launcher3.Workspace.beginDragShared(Workspace.java:1605)
	at com.android.launcher3.Workspace.beginDragShared(Workspace.java:1571)
	at com.android.launcher3.Workspace.startDrag(Workspace.java:1548)
	at com.android.launcher3.touch.ItemLongClickListener.beginDrag(ItemLongClickListener.java:76)
	at com.android.launcher3.touch.ItemLongClickListener.onWorkspaceItemLongClick(ItemLongClickListener.java:57)
	at com.android.launcher3.touch.ItemLongClickListener.lambda$nz9MSaglTImbNX-jBQmvpOY7s8M(Unknown Source:0)
	at com.android.launcher3.touch.-$$Lambda$ItemLongClickListener$nz9MSaglTImbNX-jBQmvpOY7s8M.onLongClick(Unknown Source:0)
	at android.view.View.performLongClickInternal(View.java:6756)
	at android.view.View.performLongClick(View.java:6714)
	at com.android.launcher3.CheckLongPressHelper.triggerLongPress(CheckLongPressHelper.java:136)
	at com.android.launcher3.CheckLongPressHelper.lambda$WXH_Q7mG_GcP_BvvhOplFmna0F0(Unknown Source:0)
	at com.android.launcher3.-$$Lambda$CheckLongPressHelper$WXH_Q7mG_GcP_BvvhOplFmna0F0.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6898)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by byteme-git** [comment]
I have a similar Issue but only when closing/restarting the App.

It is really sad because it makes this amazing feature effectively unusable.

Below is a short report about my version of the Issue.

#### Steps to Reproduce
1. Activate cover mode for a folder
2. Restart the Omega Launcher
Result is exactly the same as described by @izzel 

#### Device and OS
* Xiaumi Poco F1
* Android 11
* Rom: Lineage OS with microG
             18.1-20210726-microG-beryllium
* Launcher version: 0.8.0  (from F-Droid Repo)

**Comment by otakuhqz** [comment]
This issue has been fixed in the current development version. A new version will be available in the next few days.

**Comment by otakuhqz** [comment]
Fixed on v0.8.1
